ibis/expr/api.py

```diff
@@ -1274,6 +1274,21 @@ def trailing_range_window(preceding, order_by, group_by=None):
     )
 
 
+@functools.wraps(ir.Table.union)
+def union(table: ir.Table, *rest: ir.Table, distinct: bool = True):
```

Comment:
`distinct` should default to `False` to match the method.

I believe the defaults differ because `union` with `distinct=True` is not as common as `distinct=False` and also a performance footgun.

Reply:
Right, updating.